cmd/cluster/powervs/destroy.go

```diff
@@ -19,7 +19,7 @@ import (
 func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:          "powervs",
-		Short:        "Destroys a HostedCluster and its resources on PowerVS",
+		Short:        "Destroys a HostedCluster and its associated infrastructure on PowerVS.",
```

Comment:
lets limit the change to install-cnv.sh.

Reply:
ok.